imagersite/imagersite/tests.py

```diff
@@ -0,0 +1,21 @@
+from django.test import TestCase, Client
+
+
+class BasicViewTests(TestCase):
+    def test_home(self):
+        response = self.client.get('/')
+        self.assertEqual(response.status_code, 200)
+
+    def test_login(self):
+        response = self.client.get('/accounts/login/')
+        self.assertEqual(response.status_code, 200)
+
+    def test_register(self):
+        response = self.client.get('/accounts/register/')
+        self.assertEqual(response.status_code, 200)
+
+    def test_logout(self):
+        response = self.client.get('/accounts/logout/')
+        self.assertEqual(response.status_code, 200)
```

Comment:
You should also test that the HTML returned for each view is correct. That is, if you go to the home route, do you get the home page.